ci/qa/phpcs.xml

```diff
@@ -18,4 +18,9 @@
             <property name="absoluteLineLimit" value="150"/>
         </properties>
     </rule>
+
+<!--
+    Temporarily disabled because of failures in tests
+    <rule ref="vendor/slevomat/coding-standard/SlevomatCodingStandard/Sniffs/TypeHints/DeclareStrictTypesSniff.php" />
+    -->
```

Comment:
ok i'll remove it

Reply:
removed all commented out strict types

Reply:
"A lot of PHPDoc type hints are still present. Please remove them and at those points I missed type definitions."
Do you have a trickt how to detect them all?

Reply:
BTW, the empty replacedment is done by rector

Reply:
Thanks for removing the irrelevant `//declare(strict_types);` statements.

I indeed did notice some additional whitespaces being added. Maybe we can simply remove them using the PHPStorm formatter? Low priority at this point. But should be easy enough to fix.

Did a small test, and it can be done using PHPStorm

1. select a directory you want to reformat
2. In the `Code` menu select `Reformat code`
3. Choose the reformat preferences you want to apply. (I also opted to reformat the imports)
4. Bob's your brothers mother